src/app/portlets/shared/dot-content-types-listing/dot-content-types.component.ts

```diff
@@ -80,22 +79,19 @@ export class DotContentTypesPortletComponent implements OnInit {
         private pushPublishService: PushPublishService,
         private route: ActivatedRoute,
         private router: Router,
-        public dotMessageService: DotMessageService
+        public dotMessageService: DotMessageService,
+        private dotPushPublishDialogService: DotPushPublishDialogService
     ) {}
 
     ngOnInit() {
         forkJoin(
             this.dotMessageService.getMessages(this.i18nKeys),
             this.dotContentTypeService.getAllContentTypes(),
             this.dotLicenseService.isEnterprise(),
-            this.pushPublishService.getEnvironments().pipe(
-                map((environments: DotEnvironment[]) => !!environments.length),
-                take(1)
-            ),
-            this.route.data.pipe(
-                pluck('filterBy'),
-                take(1)
-            )
+            this.pushPublishService
```

Comment:
How many columns do you have setup in your editor?, is weird that this is happening.